docs/UserGuide.md

```diff
@@ -77,45 +77,73 @@ Format: `help`
 
 ### Adding a person: `add`
 
-Adds a person to the address book.
+Adds a student to Tuteez.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `add n/NAME p/PHONE_NUMBER [e/EMAIL] [a/ADDRESS] [t/TAG]…​ [l/LESSON]…​`
 
+- The **`l/` (lesson)** field should include the **day** of the week (case-insensitive) followed by the **time** in the format `HHMM-HHMM`, separated by a space.
```

Comment:
Do you think it will be clearer if we specify HHMM is in 24hr format?

Reply:
Yes that will be better! Good catch!